jenkins/values.yaml

```diff
@@ -5,7 +5,7 @@ Master:
     - workflow-aggregator:2.5
     - credentials-binding:1.16
     - git:3.9.1
```

Comment:
While you are changing this, git:3.9.1 is affected by this [CSRF Vulnerability](https://jenkins.io/security/advisory/2019-01-28/#SECURITY-1095). git:3.9.3 is their latest stable release